src/pages/leaderboard.tsx

```diff
@@ -23,7 +22,7 @@ const LeaderboardPage = (props: LeaderboardProps) => {
   return (
     <div className={styles.container}>
       <div className={styles.header}>
-        <h1>Leaderboard</h1>
+        <h1 className={styles.heading}>Leaderboard</h1>
```

Comment:
No changes needed for this PR since we need to talk this through at a meeting at some point but it might be a good idea to eventually turn this into a reusable title component for future pages.